test/backend.test.ts

```diff
@@ -99,6 +100,7 @@ describe("Tests ReactNativeBackend", () => {
       new ReactNativeBackend({
         dsn: EXAMPLE_DSN,
         enableNative: true,
+        transport: NoopTransport,
```

Comment:
Is this because of the 6.5.1 bump, or unrelated?

Reply:
@AbhiPrasad Yes, looks like it was due to this change https://github.com/getsentry/sentry-javascript/commit/ddda018d5a9bbf46437467c69608f947e4f679f2 made in 6.2.2. This line https://github.com/getsentry/sentry-javascript/blob/172e478084ec0d61b8f30efdda476e7e33e5237f/packages/browser/src/transports/fetch.ts#L73 throws as `global.fetch` is undefined.